fala/urls.py

```diff
@@ -2,7 +2,15 @@
 from django.conf import settings
 
 from django.urls import re_path
-from adviser.views import AccessibilityView, AdviserView, SearchView, PrivacyView, CookiesView
+from adviser.views import (
+    AccessibilityView,
+    AdviserView,
+    SearchView,
+    PrivacyView,
+    CookiesView,
+    robots_txt,
```

Comment:
i dont know if these need to match the style of the other views here but they feel like different pages so it doesnt seem too wrong but happy to amend if needed

Reply:
This is fine, it looks odd as you're not importing a class but a function instead.